### DISCUSSION TITLE
leaf-filtered strategy in QTree

### DISCUSSION THREAD

**Discussion by umonaca** [quasarframework/quasar#9275] [General - Components / Directives / etc]
According to the "Tick Strategy" part of QTree document, there are four tick strategies available: `strict`, `leaf`, `leaf-filtered`, `none`.  
But I am unable to see the difference between `leaf` and `leaf-filtered` in the demo.  
Can someone give an working example showing the difference of  `leaf-filtered` and `leaf`?  
Please also tell me the steps to reproduce the difference.  
Thanks!

**Comment by donnyJ42** [comment]
hope this helps: https://v0-17.quasar-framework.org/components/tree.html#Tick-Strategy
